tests/devices/test_default_qutrit.py

```diff
@@ -1440,6 +1430,11 @@ def circuit(x):
     def test_correct_state(self, tol, use_jit):
         """Test that the device state is correct after evaluating a
         quantum function on the device"""
+        from jax import numpy as jnp
+
+        if use_jit:
+            pytest.skip()
```

Comment:
Why is this test skipped with jit?

Reply:
The jitting breaks trying to get information from the  function @albi3ro made a comment about this originally. LEt me know if there is something else you'de like here.